Sources/RInAppMessaging/InAppMessagingModuleConfiguration.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+internal struct InAppMessagingModuleConfiguration {
+    let configurationURL: String?
```

Comment:
I'm wondering if we should call it `configurationURLString` as `configurationURL` seems to refer to `URL` type.